src/wp-admin/credits.php

```diff
@@ -60,7 +60,7 @@
 				);
 				?>
 				<br />
-				<a href="https://make.wordpress.org/"><?php _e( 'Get involved in WordPress.' ); ?></a>
+				<a href="https://make.wordpress.org/contribute"><?php _e( 'Get involved in WordPress.' ); ?></a>
```

Comment:
```suggestion
				<a href="https://make.wordpress.org/contribute/"><?php _e( 'Get involved in WordPress.' ); ?></a>
```
Let's add a slash here to avoid an extra redirect 🙂

Reply:
There is also another link in line 71 that needs the same adjustment.